src/app/scripts/cac/pages/cac-pages-directions.js

```diff
@@ -40,10 +51,63 @@ CAC.Pages.Directions = (function ($, _, DirectionsList, Itinerary, Settings, Uti
             var itineraries = data.plan.itineraries;
             var params = data.requestParameters;
             var itinerary = new Itinerary(itineraries[itineraryIndex], itineraryIndex, params);
+            setMapItinerary(itinerary);
             directionsListControl.setItinerary(itinerary);
         }, function (error) {
             console.log('error: ', error);
         });
+
+        function loadMap() {
+            var mapOptions = {
+                dragging: false,
+                touchZoom: false,
+                scrollWheelZoom: false,
+                doubleClickZoom: false,
+                boxZoom: false,
+                keyboard: false,
+                zoomControl: false
+            };
+            var retina = '';
+            if (window.devicePixelRatio > 1) {
+                retina = '@2x';
+            }
+            map = cartodb.L.map('directions-map', mapOptions).setView(center, zoom);
+            var basemap = cartodb.L.tileLayer('https://cartodb-basemaps-{s}.global.ssl.fastly.net/light_all/{z}/{x}/{y}' + retina + '.png', {
+                attribution: cartodbAttribution
+            });
+            basemap.addTo(map);
+        }
+
+        function setMapItinerary(itinerary) {
+            map.fitBounds(itinerary.geojson.getBounds(), {
+                padding: [30,30]
+            });
+            itinerary.geojson.addTo(map);
```

Comment:
Should fit to the geojson bounds here, instead of setting to predefined center/zoom above.